src/io/iologindata.cpp

```diff
@@ -352,10 +352,9 @@ bool IOLoginData::hasBiddedOnHouse(uint32_t guid) {
 std::forward_list<VIPEntry> IOLoginData::getVIPEntries(uint32_t accountId) {
 	std::forward_list<VIPEntry> entries;
 
-	std::ostringstream query;
-	query << "SELECT `player_id`, (SELECT `name` FROM `players` WHERE `id` = `player_id`) AS `name`, `description`, `icon`, `notify` FROM `account_viplist` WHERE `account_id` = " << accountId;
+	std::string query = fmt::format("SELECT `player_id`, (SELECT `name` FROM `players` WHERE `id` = `player_id`) AS `name`, `description`, `icon`, `notify` FROM `account_viplist` WHERE `account_id` = {}", accountId);
 
-	DBResult_ptr result = Database::getInstance().storeQuery(query.str());
```

Comment:
It's not mandatory either, but since you're messing around here, you can change Database::getInstance to g_database().